src/main/java/com/griefcraft/util/config/Configuration.java

```diff
@@ -66,7 +67,7 @@ protected Configuration(File file) {
         options.setIndent(4);
         options.setDefaultFlowStyle(DumperOptions.FlowStyle.BLOCK);
 
-        yaml = new Yaml(new SafeConstructor(), new Representer(), options);
+        yaml = new Yaml(new SafeConstructor(new LoaderOptions()), new Representer(options), options);
```

Comment:
Any way to have this continue to work for versions prior to 1.20? Use reflection?

Reply:
Alternatively, don't see a reason why this can't use Bukkit's YAML loader. Might be more work though, don't mind either way.

Reply:
Not sure if that's advisable, I feel like the better alternative would be to just shade in SnakeYAML. (Or using the Bukkit config API of course) 

Doing some hacky multiversion support there just ends up being more and complicated in the long run imo.

Reply:
Not unsupported, just less of a focus. If you take a look I've already added support for 1.20 features, LWC is still receiving the same updates it normally does.

As discussed on Discord it sounds like the best solution here is just setting the API version to 1.18, according to bStats there are not many users on 1.13 - 1.17 so it may make sense to drop those (plus, with 1.18+ we can switch to use Java 17 here as an added bonus).